artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/jgroups/JChannelManager.java

```diff
@@ -34,7 +34,7 @@
 
    private static final Logger logger = Logger.getLogger(JChannelManager.class);
 
-   private Map<String, JChannelWrapper> channels;
```

Comment:
@clebertsuconic The ChannelBroadcastEndpointFactory is not used in my code path. Instead, the JGroupsFileBroadcastEndpointFactory is used, see FileConfigurationParser line 1218 and 1251.